### PR TITLE
[160] Create & Style the Customer Signup Page using CSS

### PR DESCRIPTION
**Pull request by HalimahOladimeji**
Laptop view
Screenshot 2020-12-08 192700
Screenshot 2020-12-08 192744
Tablet view
Screenshot 2020-12-08 192520
Screenshot 2020-12-08 192602
Mobile view
Screenshot 2020-12-08 203601
Screenshot 2020-12-08 212342
Screenshot 2020-12-08 211456

### CODE REVIEW

File: submissions/HalimahOladimeji/HTML-forms/signup.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        <ul class="headerc">
+            <li><a href="../HTML-forms/index.html">Product Page</a></li>
+            <li><a href="../HTML-forms/contactus.html">Contact Us</a></li>
+            <li><a href="../HTML-forms/signup.html">Sign-up</a></li>
+        </ul>
+    </nav>
+    <div class="register">
+        <div class="secondcolumn" id="first">
+            <h1>Register your Account</h1>
+            <p>Registering an account gives you<br>access to sweet promo and events<br>available to only
```
Comment (Ireti203) [inline submissions/HalimahOladimeji/HTML-forms/signup.html:23]: Remove the breaks. Style with CSS

**Comment by HalimahOladimeji** [comment]
Corrected. 
Screenshot 2020-12-09 133258

**Review by Ireti203 — CHANGES_REQUESTED** [review]
Avoid usings breaks on your html file. Use CSS to style instead. Check for places where you used breaks and make corrections.

**Comment by HalimahOladimeji** [comment]
Noted ma. I have removed the breaks in the form too.
Screenshot 2020-12-09 134556

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**